packages/sfui/frameworks/vue/components/SfSelect/SfSelect.vue

```diff
@@ -41,19 +41,16 @@ const emit = defineEmits<{
   (event: 'update:modelValue', param: string): void;
 }>();
 
-const selected = ref(props.value);
 const chevronRotated = ref(false);
 const { isFocusVisible } = useFocusVisible();
 
-const clickHandler = () => (chevronRotated.value = true);
-const blurHandler = () => (chevronRotated.value = false);
-const keydownHandler = () => (chevronRotated.value = true);
+const onOpen = () => (chevronRotated.value = true);
+const onClose = () => (chevronRotated.value = false);
```

Comment:
Wondering, could we use `useDisclaimer` here? 😄

Reply:
will check this out, and btw its `useDisclosure`

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2933/commits/c9728e48d1f4e7d0fb36a8dbb4f0a08d8c053a69